build-tools/packages/build-tools/src/bumpVersion/createReleaseBump.ts

```diff
@@ -17,8 +17,8 @@ import * as semver from "semver";
  * Create release bump branch based on the repo state for either main or next branches,bump minor version immediately
  * and push it to `main` and the new release branch to remote
  */
-export async function createReleaseBump(context: Context, bumpTypeOverride: VersionBumpType | undefined, virtualPatch: boolean) {
-    if (context.originalBranchName === "main") {
+export async function createReleaseBump(whatToBump: MonoRepoKind, context: Context, bumpTypeOverride: VersionBumpType | undefined, virtualPatch: boolean, skipPolicyCheck = false, skipUpToDateCheck = false, skipInstall=false) {
```

Comment:
This exceeds our max line length right?
Also:
```suggestion
export async function createReleaseBump(whatToBump: MonoRepoKind, context: Context, bumpTypeOverride: VersionBumpType | undefined, virtualPatch: boolean, skipPolicyCheck = false, skipUpToDateCheck = false, skipInstall = false) {
```

Reply:
Fixed.